std/uni.d

```diff
@@ -8131,6 +8185,7 @@ private auto toCaser(alias indexFn, uint maxIdx, alias tableFn, alias asciiConve
         {
             @property auto save()
             {
+                import std.range.primitives : save;
```

Comment:
Do we really need it given the top-level std.range.primitives import, imo this is getting too picky for front/empty/popFront